mailbox/plugin/deleted-messages-vault/src/main/java/org/apache/james/vault/DeletedMessageVaultHook.java

```diff
@@ -146,9 +146,7 @@ public final int hashCode() {
     private Flux<DeletedMessageMailboxContext> groupMetadataByOwnerAndMessageId(DeleteOperation deleteOperation) {
```

Comment:
we don't group by owner anymore. No ?

Reply:
We were grouping by messageId.

The ticket & PR description says:

```
Drawback: if a message is referenced in several mailboxes at once, we expect it to be inserted several time in the (indempotent) deleted message vault, leading to a performance penalty. However this is a rare unexpected corner case, and it looks like premature optimization.
```

I'm waiting CI feedback to see if I broke something, but if I don't then yes, we will likely not do the second (messageId) groupBy anymore.

Reply:
Sorry what i meant is that we should rename the name of the method now.